tests/Integration/Database/MultiDatabaseTest.php

```diff
@@ -0,0 +1,157 @@
+<?php
+
+namespace Integration\Database;
+
+use Tempest\Container\Exceptions\CannotResolveTaggedDependency;
+use Tempest\Database\Config\DatabaseDialect;
+use Tempest\Database\Config\MysqlConfig;
+use Tempest\Database\Config\SQLiteConfig;
+use Tempest\Database\DatabaseInitializer;
+use Tempest\Database\Id;
+use Tempest\Database\Migrations\CreateMigrationsTable;
+use Tempest\Database\Migrations\MigrationManager;
+use Tests\Tempest\Fixtures\Migrations\CreatePublishersTable;
+use Tests\Tempest\Fixtures\Modules\Books\Models\Publisher;
+use Tests\Tempest\Integration\FrameworkIntegrationTestCase;
+use Tests\Tempest\Integration\TestingDatabaseInitializer;
+
+use function Tempest\Database\query;
+
+final class MultiDatabaseTest extends FrameworkIntegrationTestCase
+{
+    protected function setUp(): void
+    {
+        parent::setUp();
+
+        if (file_exists(__DIR__ . '/Fixtures/main.sqlite')) {
```

Comment:
Missed the opportunity to use the new `Filesystem` utils :p

Reply:
Aah I should refactor that!